frigate/record/cleanup.py

```diff
@@ -60,7 +61,9 @@ def truncate_wal(self) -> None:
             db.execute_sql("PRAGMA wal_checkpoint(TRUNCATE);")
             db.close()
 
-    def expire_review_segments(self, config: CameraConfig, now: datetime) -> None:
+    def expire_review_segments(
+        self, config: CameraConfig, now: datetime.datetime
```

Comment:
Either this or change top level `import datetime` to `from datetime import datetime`